components/CovidMap.js

```diff
@@ -0,0 +1,702 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tabletop from 'tabletop';
+import MarkerClusterer from '@google/markerclustererplus';
+
+const mapStyle = {
+  height: '400px', 
+  width: '100%',
+};
+
+const clusterStyleFirstSmall = {
+  width: '30', 
+  height: '30', 
+  className: 'custom-clustericon-first'
+}
+
+const clusterStyleFirstMed = {
+  width: '35', 
+  height: '35', 
+  className: 'custom-clustericon-first'
+}
+
+const clusterStyleFirstLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-first'
+}
+
+const clusterStyleSecondSmall = {
+  width: '30', 
+  height: '30', 
+  className: 'custom-clustericon-second'
+}
+
+const clusterStyleSecondMed = {
+  width: '35', 
+  height: '35', 
+  className: 'custom-clustericon-second'
+}
+
+const clusterStyleSecondLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-second'
+}
+
+const clusterStyleThirdSmall = {
+  width: '30', 
+  height: '30', 
+  className: 'custom-clustericon-third'
+}
+
+const clusterStyleThirdMed = {
+  width: '35', 
+  height: '35', 
+  className: 'custom-clustericon-third'
+}
+
+const clusterStyleThirdLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-third'
+}
+
+const formStyle = {
+  width: '50%',
+  float: 'left', 
+}
+
+const legendStyle = {
+  width: '50%', 
+  float: 'right',
+}
+
+const legendItemStyle = {
+  width: '100%',
+  marginBottom: '5px',
+}
+
+const legendTextStyle = {
+  width: '90%',
+  height: '20px',
+  float: 'left',
+  textAlign: 'right',
+}
+
+const legendIconStyle = {
+  minWidth: '30px',
+  height: '20px',
+  float: 'right',
+  display: 'inline-block',
+}
+
+const legendIconFirst = {
+  background: '#0B5D93',
```

Comment:
Can we replace this with one of our global variables? They live [here](https://github.com/texas-justice-initiative/website-nextjs/blob/master/data/chart_colors.js) and can be imported. I don't think the purple you are using is currently a part of our color palette, can you add it into our global file as well?

@nholden on a side note, I think it might be worthwhile for us to spend some time updating our css variables to be a bit more robust. I know we wanted to at least build out some standard spacers but never got to it prior to launch.

Reply:
I agree that would be better. I used an online tool to 'blend' the red and blue used in the globals, I can add it to the global css file.